hoomd/mpcd/integrate.py

```diff
@@ -73,6 +75,72 @@ class Integrator(_MDIntegrator):
     The MD particles can be read at any time step because their positions
     are updated every step.
 
+    .. rubric:: Examples:
+
+    .. invisible-code-block: python
+
+        simulation = hoomd.util.make_example_simulation(mpcd_types=["A"])
+
+    MPCD integrator for pure solvent.
+
+    .. code-block:: python
+
+        stream = hoomd.mpcd.stream.Bulk(period=1)
+        collide = hoomd.mpcd.collide.StochasticRotationDynamics(period=1, angle=130)
+        integrator = hoomd.mpcd.Integrator(
+            dt=0.1,
+            streaming_method=stream,
+            collision_method=collide,
+            solvent_sorter=hoomd.mpcd.tune.ParticleSorter(trigger=20))
+        simulation.operations.integrator = integrator
+
+    MPCD integrator with solutes.
+
+    .. code-block:: python
+
+        dt_md = 0.005
+        dt_mpcd = 0.1
+        md_steps_per_collision = numpy.round(dt_mpcd / dt_md).astype(int)
```

Comment:
```suggestion
        dt_md = 0.005
        md_steps_per_collision = 20
        dt_mpcd = dt_md * md_steps_per_collision
```

...unless there's a better reason to set `dt_mpcd` explicitly

Reply:
Formulas for the properties of the MPCD solvent depend on `dt_mpcd` so it is sort of the more natural control variable, but the point is well taken that it still needs to be a multiple of `dt_md`.

Reply:
With this change, we can actually just remove `dt_mpcd`. I added a trailing comment to `md_steps_per_collision` clarifying this makes `dt_mpcd = 0.1`.